contrib/platform/test/com/sun/jna/platform/win32/VersionUtilTest.java

```diff
@@ -0,0 +1,42 @@
+/* This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import java.io.File;
+
+import com.sun.jna.platform.win32.VerRsrc.VS_FIXEDFILEINFO;
+
+import junit.framework.TestCase;
+
+public class VersionUtilTest extends TestCase {
+
+    public static void main(String[] args) {
+        junit.textui.TestRunner.run(VersionUtilTest.class);
+    }
+
+    public void testGetFileVersionNumbers() {
+        File file = new File(System.getenv("SystemRoot"), "regedit.exe");
+        assertTrue(file.exists());
+
```

Comment:
Be more specific:
```java
assertTrue("Test file not found: " + file, file.exists());
```

This may seem superfluous since it is "obvious" that you are looking for _regedit.exe_. However, what if the test system is mis-configured and _SystemRoot_ is not defined or defined incorrectly ? The "message-less" assertion would not provide enough information to diagnose the failure. Failing assertions should provide as much information as possible to enable us to diagnose the problem.